docs/guide/adapters/emberznet.md

```diff
@@ -112,7 +112,24 @@ With external antenna.
 
 
 ### Hybrid (USB + Network)
+<details>
+<summary>SMLIGHT SLZB-06M EFR32MG21 Zigbee Ethernet USB POE WiFi LAN adapter</summary>  
+
+  
+SMLIGHT SLZB-06M is an EFR32MG21 chip-based adapter. Powerful, tiny, modern design, developed for Zigbee2MQTT, PoE supported, pre-flashed, and ready-to-use Zigbee adapter.
+It supports Zigbee 3.0, Thread, and Bluetooth with Ethernet, LAN, USB, and WiFi connections.  
+[![](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-06m-main-title-small.jpg)](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-06m-main-title.jpg)  
+  
+**For optimal performance with Ember, ensure your Zigbee firmware is above version 20240510 (requires Core firmware to a version higher than 2.2.0)**.  
+Currently, please avoid using Ember firmware with a Wi-Fi connection, as it can cause some interference.  
```

Comment:
Can we go with 
```markdown
<img src="https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-06m-main-title.jpg" width="200" />
```
for the image, to avoid potential scrolling issues (all images constrained to the same width no matter what).

```markdown
**For optimal performance, ensure your firmware is at least version 20240510 (requires Core firmware version 2.2.0 or higher).
Not suitable for WiFi or other high latency connections.**
```

Though I'm thinking we should add a link for each adapter with the "currently recommended firmware" in a more generic fashion? Something like this in the list for each adapter:

```markdown
* [Recommended firmware](#firmwarelinkhere)
```

_I've seen far too many wrong baudrate flashed without the appropriate setting set in Z2M in last few days, resulting in Z2M not starting, that would at least provide a link for "latest stable", with default baudrate._

Reply:
> Can we go with

thanks. done here https://github.com/Koenkk/zigbee2mqtt.io/pull/2747/commits/c52211e849887d6fe08204b69ac44dc7567ec20f